src/tests/ftest/control/dmg_support_collect_log.py

```diff
@@ -0,0 +1,51 @@
+"""
+  (C) Copyright 2023 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+from support_test_base import SupportTestBase
+
+
+class DmgSupportCollectLogTest(SupportTestBase):
+    # pylint: disable=too-many-ancestors
+    """Test Class Description:Verify the dmg support collect-log command.
+
+    :avocado: recursive
+    """
+
+    def test_dmg_support_collect_log(self):
+        """JIRA ID: DAOS-10625
+
+        Test Description:
+            Test that dmg support collect-log command completes successfully.
+
+        :avocado: tags=all,daily_regression
+        :avocado: tags=hw,medium
+        :avocado: tags=basic,control,support,dmg
+        :avocado: tags=DmgSupportCollectLogTest,test_dmg_support_collect_log
+        """
+        self.log_hosts = self.hostlist_servers
+        # Create the custom log data which will be collected via support collect-log,
+        # Later verify the dame data file is archived as part of collection.
+        self.create_custom_log("Support_Custom_Dir")
+
+        # Run dmg support collect-log with --extra-logs-dir,
+        # copy logs to folder with command option --target-folder
+        # Enable archive mode.
+        result = self.dmg.support_collect_log(extra_logs_dir=self.custom_log_dir,
+                                              target_folder=self.target_folder,
+                                              archive=True)
+        status = result["status"]
+        self.assertEqual(status, 0, "bad return status from dmg support collect-log")
```

Comment:
`dmg.support_collect_log` uses `_get_json_result`, which raises an exception by default unless you set `dmg.exit_status_exception = False`. So you'll probably never get a result where status is not 0

Reply:
Ok, so result.passed is preferred check instead of status?

Reply:
`result.passed` is for when using `run_remote`.
`dmg.support_collect_log` uses `_get_json_result`, which returns a dictionary of the json result. But `_get_json_result` will raise an exception if `result["status"] != 0`. Which means the `self.assertEqual(status, 0, ...)` will never happen, because `_get_json_result` would have already raised an exception. You could just remove the status check here :)